packages/cli/src/types/dataset.ts

```diff
@@ -0,0 +1,47 @@
+interface DatasetMetadata {
+  description: string;
+  url: string;
+  version: string;
+  year: number;
+  contributor: string;
+  date_created: string;
+}
+
+export interface ImageSchema {
+  license: number;
+  file_name: string;
+  coco_url: string;
+  height: number;
+  width: number;
+  id: number;
+}
+
+interface DataSetJSONLicense {
+  url: string;
+  id: number;
+  name: string;
+}
+
+export interface DataSetJSONAnnotation {
+  id: number;
+  image_id: number;
+  category_id: number;
+  segmentation: any[];
+  iscrowd: number;
+  area: string;
+  bbox: string;
+}
+
+export interface DataSetJSONCategory {
+  id: number;
+  name: string;
+  supercategory: string;
+}
+
+export interface DataSetJSON {
+  info: DatasetMetadata;
+  images: ImageSchema[];
+  licenses: DataSetJSONLicense[];
+  annotations: DataSetJSONAnnotation[];
+  categories: DataSetJSONCategory[];
+}
```

Comment:
It's good to rename all the interface names as:

1. "DataSet" to "Dataset"
2. Remove the duplicated JSON alphabets.

Reply:
done.